crates/ruma-events/tests/it/ui/01-content-sanity-check.rs

```diff
@@ -9,4 +9,10 @@ pub struct MacroTestContent {
     pub url: String,
 }
 
-fn main() {}
+fn main() {
+    use ruma_events::{BooleanType, StateEventContent, StaticEventContent};
+
+    assert_eq!(MacroTestContent::TYPE, "m.macro.test");
+    assert!(!<MacroTestContent as StaticEventContent>::IsPrefix::as_bool());
+    assert_eq!(MacroTestContent { url: "foo".to_owned() }.event_type().to_string(), "m.macro.test");
```

Comment:
```suggestion
    assert_eq!(MacroTestContent { url: "foo".to_owned() }.event_type(), "m.macro.test");
```

can the event type not be compared with a string literal directly?

Reply:
No, none of the `*EventType` enums can be compared to a string.